settle-bot/Dockerfile.liquidator

```diff
@@ -0,0 +1,7 @@
+# Dockerfile for keeper service in Heroku
```

Comment:
probably also want to copy paste stuff to push settle bot to gcr, see ci-docker-publish.yml, if it doesnt work out of the box, we could ask pan of course